webapp/src/app/shared/widgets/event-widget/event-registry/ShowChoice.ts

```diff
@@ -73,7 +73,8 @@ export class ShowChoice extends AbstractEvent<ShowChoiceData> {
 			if (!child.events) {
 				console.error('wtf', this);
 			}
-			out[index as keyof ShowChoiceData] = child.events.map(v => v.export());
+			// @ts-ignore
```

Comment:
Why was this changed, especially in an PR that modifies entity? Also, before adding exceptions like `@ts-ignore` the type should be changed to fit the demand.

Reply:
Linters pass, but typescript itself complains after updating. 
Problem is the type only matches for the number props not the other and i couldn't figure out how to fix that and still have a somewhat readable code

Reply:
Defining the indexer `[index: number]: AbstractEvent<any>[];` on `ShowChoiceData` should be able to solve that problem.

Reply:
That's a good idea, didn't thought of that